content/about/about_people.rst

```diff
@@ -590,6 +590,33 @@ https://github.com/ElijahCaine
 
   <br/>
 
+`Levi Willmeth`_
+----------------
+
+.. image:: /images/lwillmeth.jpg
+    :scale: 50%
+    :align: right
+    :alt: Levi Willmeth
+
+.. class:: no-breaks
+
+  **OSL Academy Intern**
+
+Levi joined the OSL as a student intern in the summer of 2016. He is married
+and has two energetic toddlers who help ensure he doesn't sleep too much.
+Levi enjoys playing bike polo, flying RC aircraft, looking thoughtful during 
```

Comment:
"looking thoughtful in photographs" (when we figure this out, it'll be a hit. Trust me, I'm a writer).